siberi-library/src/main/java/com/mercari/siberi/Siberi.java

```diff
@@ -43,6 +49,10 @@ public void run() {
                     int variant = object.optInt("variant");
                     JSONObject metaData = object.optJSONObject("metadata");
                     sStorage.insert(test, variant, metaData);
+
+                    if (callback != null) {
+                        callback.hasSetExperimentContents();
```

Comment:
I'm not sure if this is the right place to set a callback. if `dataArray.length()` is 100, `hasSetExperimentContents ` will be called 100 times?